image/preseed/preseed_linux.go

```diff
@@ -559,6 +588,26 @@ func prepareClassicChroot(preseedChroot string) (*targetSnapdInfo, func(), error
 	}
 	addCleanup(unmountSnapd)
 
+	// because of the way snapd snap is built, we need the
+	// 'current' symlink to exist when the snapd binary is
+	// invoked, so that any runtime libraries will be correctly
+	// resolved, so bind-mount a snapd snap at a side location set
```

Comment:
...and set up? The comment is a bit confusing

Reply:
Would this be better? "so we bind-mount the snapd snap at a side location, and create the symlink 'current' to point to it"

Reply:
yeah, that would be perfect, thank you

Reply:
Done.